ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -893,6 +893,7 @@ class EclCmdRun : public EclCmdWithEclTarget
             break;
         default:
             fprintf(stderr, "%s %s\n", respwuid.str(), resp->getState());
+            ret = 1;
```

Comment:
@richardkchapman I did debate a bit about submitted, compiling, and compiled.  But if I return success for those preliminary states, perhaps I should output something like 
`fprintf(stderr, "Timed out waiting for %s to complete, workunit may still be running.\n", wuid.str());`